SQL/0000-00-00-schema.sql

```diff
@@ -2044,3 +2049,20 @@ CREATE TABLE `candidate_consent_history` (
   `EntryStaff` varchar(255) DEFAULT NULL,
   CONSTRAINT `PK_candidate_consent_history` PRIMARY KEY (`CandidateConsentHistoryID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `project_external` (
+  `ProjectExternalID` int(4) NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) UNIQUE DEFAULT NULL,
+  PRIMARY KEY (`ProjectExternalID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `candidate_project_external_rel` (
```

Comment:
I find the underscore in the table name makes this look a little like a trinary relationship table, but I guess there's nothing to do for that given our existing naming standards.